packages/react-components/src/components/HorizontalGallery.tsx

```diff
@@ -107,13 +108,15 @@ const HorizontalGalleryNavigationButton = (props: {
   styles: IStyle;
   onClick?: () => void;
   disabled?: boolean;
+  ariaLabel?: string;
```

Comment:
possible to provide `aria-label` directly without declaring a conflicting new prop `ariaLabel` ?
Another alternative would be to have `HorizontalGalleryNavigationButton` extend `HTMLButton` and it should be able to accept `aria-label` directly without a new prop.